frontend/catalyst/jit.py

```diff
@@ -591,8 +591,9 @@ def closure(qnode, *args, **kwargs):
         with Patcher(
             (qml.QNode, "__call__", closure),
             # !!! TODO: fix jax.scipy numerical failures with properly fetched lapack calls
-            # As of now, we raise a warning prompting the user to use a callback with catalyst.accelerate()
-            # https://app.shortcut.com/xanaduai/story/70899/find-a-system-to-automatically-create-a-custom-call-library-from-the-one-in-jax
+            # As of now, we raise a warning prompting the user to use a callback
+            # with catalyst.accelerate()
+            # https://app.shortcut.com/xanaduai/story/70899/find-a-system-to-automatically-create-a-custom-call-library-from-the-one-in-jax  # pylint: disable=line-too-long
```

Comment:
I think you might want to reverse this diff to avoid conflicts with #1097

Reply:
good point, I'll leave jit.py unchanged in this PR